### ISSUE TITLE
Last played show clear logo on OSD

### ISSUE THREAD

**Issue by IG2022** [jurialmunkey/skin.arctic.horizon.2#88]
Hi Jurial,

I've noticed a small bug with clear logos in the OSD. If I play an episode or movie with clear logo available for the title in the OSD the clear logo displays correctly as the title on the OSD. If I then play a movie or episode that does not have a clear logo available the clear logo from the last played show or movie pops up in the OSD for the title rather than falling back to regular text and showing the correct movie or episode title. This isn't a huge issue but can be a little distracting as the clear logo is sometimes not correct and can get confusing. 

I'm not sure if this is a small fix but I just wanted to bring it to your attention.  

Thanks again for all your hard work I'm really enjoying the skin.

**Comment by xyzfre** [comment]
I have also experienced this occasionally I actually thought it was something on my end till I saw this post

**Comment by jurialmunkey** [comment]
Yeah I can confirm. Occasionally TMDbH doesn't clear out the old logo. I've had it happen with shows that do have logos, so I'm not 100% on what the actually underlying cause is.

**Comment by IG2022** [comment]
> Yeah I can confirm. Occasionally TMDbH doesn't clear out the old logo. I've had it happen with shows that do have logos, so I'm not 100% on what is actually the underlying cause.

That's interesting, I was thinking the cause was the show or movie not having a clear logo, I guess not. It's not a huge deal it doesn't happen that often anyway.
